src/test/java/com/znsio/teswiz/screen/android/vodqa/VodqaScreenAndroid.java

```diff
@@ -218,4 +221,57 @@ public boolean isDoubleTapSuccessful() {
         visually.check(SCREEN_NAME,"Double Tap Successful Message", Target.region(byDoubleTapSuccessfulXpath));
         return driver.isElementPresent(byDoubleTapSuccessfulXpath);
     }
+
+    @Override
+    public VodqaScreen navigateToPhotoView() {
+        LOGGER.info("Navigate to Photo View Section");
+        driver.waitTillElementIsVisible(byDoubleTapSuccessfulXpath);
+        driver.waitTillElementIsVisible(byPhotoViewElementXpath).click();
+        return this;
+    }
+
+    @Override
+    public VodqaScreen pinchAndZoomInOnAnElement() {
+        LOGGER.info("Zoomed In Image element");
+        visually.check(SCREEN_NAME,"Zoomed In Image element",Target.window());
+        driver.waitTillElementIsVisible(byDoubleTapSuccessfulXpath);
+        driver.pinchAndZoomIn(driver.waitTillElementIsVisible(byImageElementXpath));
+        return this;
+    }
+
+    @Override
+    public VodqaScreen pinchAndZoomOutOnAnElement() {
+        LOGGER.info("Zoomed Out Image element");
+        visually.check(SCREEN_NAME,"Zoomed Out Image element",Target.window());
+        driver.waitTillElementIsVisible(byDoubleTapSuccessfulXpath);
+        driver.pinchAndZoomOut(driver.waitTillElementIsVisible(byImageElementXpath));
+        return this;
+    }
+
+    @Override
+    public boolean isPinchAndZoomInSuccessful(Dimension initialElementDimension) {
+
+        LOGGER.info("Validate pinch and zoom in on element is successful");
+        Dimension actualElementDimension = driver.getDimension(driver.waitTillElementIsVisible(byImageElementXpath));
+        if ((initialElementDimension.width * initialElementDimension.height) > (actualElementDimension.width * actualElementDimension.height)) {
+            return true;
+        }
+        return false;
+    }
+
+    @Override
+    public boolean isPinchAndZoomOutSuccessful(Dimension initialElementDimension) {
+
+        LOGGER.info("Validate pinch and zoom out on element is successful");
+        Dimension actualElementDimension = driver.getDimension(driver.waitTillElementIsVisible(byImageElementXpath));
+        if ((initialElementDimension.width * initialElementDimension.height) < (actualElementDimension.width * actualElementDimension.height)) {
+            return true;
+        }
+        return false;
```

Comment:
you simplify this by removing these lines and writing the below line:
return (initialElementDimension.width * initialElementDimension.height) < (actualElementDimension.width * actualElementDimension.height);

Reply:
this is good idea, updated the same